### DISCUSSION TITLE
Google Cloud: Why is bucket required to upload images?

### DISCUSSION THREAD

**Discussion by darkn3rd** [flatcar/Flatcar#806] [General]
In the instructions for uploading an image ([ref](https://www.flatcar.org/docs/latest/installing/cloud/gcp/#uploading-an-image)), we need to create a bucket.  I was just wondering why a bucket is required?

Would the below process not work? 

```bash
gcloud compute images create "flatcar-stable" \
  --source-image-family "flatcar-stable" \
  --source-image-project "kinvolk-public"
```

**Comment by jepio** [comment]
The process you posted would work. The bucket is required if you want to be independent of the Flatcar publishing process, in that case you can get the image the image from the flatcar servers (https://stable.release.flatcar-linux.net/) and need to upload it to a bucket first.
